tests/utils/testTypes.ts

```diff
@@ -0,0 +1,8 @@
+export enum BlockType {
```

Comment:
I've tried using from `src/utils/types.ts`, but the compiler doesn't seem to like the `.vue` files imported in `src/utils/types.ts` - I'm not too sure what exactly is going wrong, but if I had to guess it may be related to some typescript problems with playwright (like how we need to append the imports with `.ts` for playwright to work (https://playwright.dev/docs/test-typescript#typescript-with-esm)). I'll try looking more into this tomorrow.

```
Running 3 tests using 3 workers
Error: C:\Users\david\Documents\lotion\src\components\blocks\TextBlock.vue: Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>? (6:0)

  4 | </template>
  5 |
    | ^
  7 | import { PropType } from 'vue'
  8 | import { Block } from '@/utils/types'
  9 | import Editor from '../elements/Editor.vue'

    at async Promise.all (index 0)


  3 skipped
  1 error was not a part of any test, see above for details
```

Reply:
Concluding this for posterity: Seems to be a Playwright-related issue with importing .vue files. Will use a separate utils file for now.